src/ukify/ukify.py

```diff
@@ -278,6 +292,27 @@ def parse_arg(cls, s):
 
         return cls.create(name, contents)
 
+    @classmethod
+    def parse_text_arg(cls, s):
```

Comment:
This name is strange. It's not parsing a "text", it's parsing a section name and type. `parse_section_name_and_type`? Also, this should not be a class method. It doesn't use the class for anything. (`parse_arg` is a class method because it returns a class instance.)